src/library/scala/util/Random.scala

```diff
@@ -43,11 +44,27 @@ class Random(val self: java.util.Random) extends AnyRef with Serializable {
    */
   def nextDouble(): Double = self.nextDouble()
 
+  /** Returns the next pseudorandom, uniformly distributed double value
+   *  between min (inclusive) and max (exclusive) from this random number generator's sequence.
+   */
+  def between(minInclusive: Double, maxExclusive: Double): Double = ThreadLocalRandom.current().nextDouble(minInclusive, maxExclusive)
```

Comment:
one of the whole points of `java.util.Random` and `scala.util.Random` is to always give the same series of pseudorandom numbers when seeded with the same seed. they are PRNGs, not true RNGs. and yes, testing (not just of the RNG itself, either, but of anything that uses it) is one of the reasons you might want this. another is that sometimes when doing scientific work it's important that results be reproducible.